test/AutoRest.TestServerLowLevel.Tests/url.cs

```diff
@@ -0,0 +1,120 @@
+
+        [Test]
+        public Task UrlPathsStringBase64Url() => TestStatus(async (host, key) => await new PathsClient(key, host).Base64UrlAsync( Encoding.UTF8.GetBytes("lorem")));
+
+        [Test]
+        public Task UrlPathsByteEmpty() => TestStatus(async (host, key) => await new PathsClient(key, host).ByteEmptyAsync());
+
+        [Test]
+        public Task UrlPathsByteMultiByte() => TestStatus(async (host, key) => await new PathsClient(key, host).ByteMultiByteAsync( TestConstants.ByteArray));
+
+        [Test]
+        public void UrlByteNullAsync()
+        {
+            Assert.ThrowsAsync<ArgumentNullException>(async () => await new PathsClient(null, null).ByteNullAsync(null));
+        }
+
+        [Test]
+        public Task UrlPathsDateValid() => TestStatus(async (host, key) => await new PathsClient(key, host).DateValidAsync());
+
+        [Test]
+        public Task UrlPathsDateTimeValid() => TestStatus(async (host, key) => await new PathsClient(key, host).DateTimeValidAsync());
+
+        [Test]
+        public Task UrlPathsLongPositive() => TestStatus(async (host, key) => await new PathsClient(key, host).GetTenBillionAsync());
+
+        [Test]
+        public Task UrlPathsIntUnixTime() => TestStatus(async (host, key) => await new PathsClient(key, host).UnixTimeUrlAsync( DateTimeOffset.FromUnixTimeSeconds(1460505600L).UtcDateTime));
+
+        [Test]
+        public Task UrlPathsIntNegative() => TestStatus(async (host, key) => await new PathsClient(key, host).GetIntNegativeOneMillionAsync());
+
+        [Test]
+        public Task UrlPathsIntPositive() => TestStatus(async (host, key) => await new PathsClient(key, host).GetIntOneMillionAsync());
+
+        [Test]
+        public Task UrlPathsBoolTrue() => TestStatus(async (host, key) => await new PathsClient(key, host).GetBooleanTrueAsync());
+
+        [Test]
+        public Task UrlPathsBoolFalse() => TestStatus(async (host, key) => await new PathsClient(key, host).GetBooleanFalseAsync());
+
+        [Test]
+        public Task UrlPathsLongNegative() => TestStatus(async (host, key) => await new PathsClient(key, host).GetNegativeTenBillionAsync());
+
+        [Test]
+        public Task UrlPathsFloatPositive() => TestStatus(async (host, key) => await new PathsClient(key, host).FloatScientificPositiveAsync());
+
+        [Test]
+        public Task UrlPathsFloatNegative() => TestStatus(async (host, key) => await new PathsClient(key, host).FloatScientificNegativeAsync());
+
+        [Test]
+        public Task UrlPathsDoubleNegative() => TestStatus(async (host, key) => await new PathsClient(key, host).DoubleDecimalNegativeAsync());
+
+        [Test]
+        public Task UrlPathsDoublePositive() => TestStatus(async (host, key) => await new PathsClient(key, host).DoubleDecimalPositiveAsync());
+
+        public override IEnumerable<string> AdditionalKnownScenarios { get; } = new string[] {"UrlPathsBoolFalse",
```

Comment:
I think we can delete this. We deleted the coverage tracking funcitonality.

Reply:
Want me to nuke it from all tests or just my new ones?

Reply:
Let's nuke it from everytwhere if it's not too much work.